src/js/techreport/utils/data.js

```diff
@@ -62,11 +62,13 @@ const parsePageWeightData = (metric, date) => {
       ...submetric,
       desktop: {
         ...submetric.desktop,
+        median_bytes_formatted: submetric.desktop.median_bytes > 1024 ? `${Math.round(submetric.desktop.median_bytes / 1024)} KB` : submetric.desktop.median_bytes > 1048576 ? `${Math.round(submetric.desktop.median_bytes / 1048576)} MB` : `${submetric.desktop.median_bytes} bytes`,
```

Comment:
This will always show KB even when it's > 1 MB

```suggestion
        median_bytes_formatted: submetric.desktop.median_bytes > 1048576 ? `${Math.round(submetric.desktop.median_bytes / 1048576)} MB` : submetric.desktop.median_bytes > 1024 ? `${Math.round(submetric.desktop.median_bytes / 1024)} KB` : `${submetric.desktop.median_bytes} bytes`,
```